functions/venue.js

```diff
@@ -310,6 +307,7 @@ const createVenueData_v2 = (data, context) => {
     rooms: [],
     createdAt: Date.now(),
     updatedAt: Date.now(),
+    parentId: data.parentId || "",
```

Comment:
Do we actually need this fallback as an empty string? Looks like it's an optional prop

Reply:
Optional for the form doesn't mean it should be optional for the DB, might be prudent to have that field in there even if with empty string value

Reply:
For now it is, because the same function is used to create worlds.